File: tugas/php/Disk.php
<?php

class Disk{
    // atribut
    private $type, $capacity, $price;

    // constructor
    public function __construct($type = "-", $capacity = 0, $price = 0){
        $this->type = $type;
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // setter n getter
    public function setType($type){
        $this->type = $type;
    }

    public function getType(){
        return $this->type;
    }

    public function setCapacity($capacity){
        $this->capacity = $capacity;
    }
    
    public function getCapacity(){
        return $this->capacity;
    }
    
    public function setPrice($price){
        $this->price = $price;
    }

    public function getPrice(){
        return $this->price;
    }
}

File: tugas/python/Disk.py
class Disk():

    # constructor
    def __init__(self, type="-", capacity=0, price=0):
        self.__type = type
        self.__capacity = capacity
        self.__price = price

    # setter n getter
    def setType(self, type):
        self.__type = type

    def getType(self):
        return self.__type

    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price